### PR TITLE
fly polish

### PR DESCRIPTION
**Pull request by skambashi**

### CODE REVIEW

File: src/objects/enemy.lua [hunk 1]
```diff
@@ -8,6 +8,7 @@ function Enemy:init(objects, x, y, player)
     Object.init(self, objects, x, y)
     self.player = player
     self.locked = false
+    self.passive = true
```
Comment (dvdfu) [inline src/objects/enemy.lua]: Are all combinations of locked and passive possible? Would this be better as `self.state = 'locked'`, etc?

File: src/objects/enemy.lua [hunk 2]
```diff
@@ -26,6 +27,9 @@ function Enemy:update(dt)
     if not self.locked then
         delta = self.player:getPosition() - self:getPosition()
 
+        if delta:len() > 500 then return
```
Comment (dvdfu) [inline src/objects/enemy.lua]: Class var constant

File: src/objects/enemy.lua (hunk 1 above)
Comment (skambashi) [inline src/objects/enemy.lua]: `passive` legit only for debugging

File: src/objects/enemy.lua (hunk 2 above)
Comment (skambashi) [inline src/objects/enemy.lua]: yeah good point

File: src/objects/enemy.lua (hunk 1 above)
Comment (dvdfu) [inline src/objects/enemy.lua]: 😬

**Pull request merged**